linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml

```diff
@@ -351,7 +351,7 @@
 
 - name: "Append ip_addresses and node_data to topology_outputs_libvirt_nodes"
```

Comment:
Can you update the name to include '(remote)' or somesuch in it to differentiate from the play just above it?